### DISCUSSION TITLE
Need p99, p95 support for keda cloudwatch

### DISCUSSION THREAD

**Discussion by phanisowjanyavutukuri** [kedacore/keda#4805] [Q&A / Need Help]
Hi,

When I see the query/metrics in cloudwatch with p99 i am able to get the values, but if I use same thing in keda metricStat getting this error:
keda-operator  error parsing cloudwatch metadata: metricStat 'p99' is not one of [SampleCount Average Sum Minimum Maximum]

could you please help here

**Comment by JorTurFer** [comment]
Maybe the scale implementation doesn't support that feature yet. Are you willing to add it?
